src/common.c

```diff
@@ -20,155 +20,166 @@
    and the latter can be found in base64.[ch] */
 const char itoa64[64] = BASE64_CRYPT;
 unsigned char atoi64[0x100];
-const char itoa16[16]  = HEXCHARS_lc;
+const char itoa16[16] = HEXCHARS_lc;
 const char itoa16u[16] = HEXCHARS_uc;
 
+
 unsigned char atoi16[0x100], atoi16l[0x100], atoi16u[0x100];
 
 static int initialized = 0;
 
 void common_init(void)
 {
-	const char *pos;
+    const char *pos;
 
-	if (initialized) return;
+    if (initialized) return;
 
-	memset(atoi64, 0x7F, sizeof(atoi64));
-	for (pos = itoa64; pos <= &itoa64[63]; pos++)
-		atoi64[ARCH_INDEX(*pos)] = pos - itoa64;
+    memset(atoi64, 0x7F, sizeof(atoi64));
+    for (pos = itoa64; pos <= &itoa64[63]; pos++)
+        atoi64[ARCH_INDEX(*pos)] = pos - itoa64;
```

Comment:
You are introducing lots and lots of whitespace violations. Did you review your own commit at all?

Reply:
@magnumripper I will correct that. My IDEA.

Reply:
Most importantly we indent with tabs, not spaces. It doesn't matter how many spaces you'd use, it's still not correct.